html/partials/unmanaged_files.html.haml

```diff
@@ -0,0 +1,29 @@
+- if unmanaged_files && unmanaged_files.files.length > 0
+  %div
+    %a.scope_anchor{:id => "unmanaged_files"}
+    #unmanaged_files_container.scope{"data-scope" => "unmanaged_files"}
+      - count = unmanaged_files.files.length
+      = render_partial "scope_header",  { |
```

Comment:
Don't use the `|` character to split up lines. Wrap on commas or extract code into helper.